src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -111,6 +111,14 @@ public String getFetchWaitMaxMs() {
         return getString("kafka.fetch.wait.max.ms");
     }
 
+    public String getKafkaSchemaUrl() {
```

Comment:
1. I though each time you add a new parameter, you need to add them to properties file as well, otherwise you will see a ParseException;

2. Those parameters are not really kafka parameters, they probably should be named: 'avro.schema.url' ...